confluent-kafka-mixin/dashboards/confluent-kafka-overview.json

```diff
@@ -1767,18 +1784,19 @@
         "query": "prometheus",
         "queryValue": "",
         "refresh": 1,
-        "regex": "",
+        "regex": "(?!grafanacloud-usage|grafanacloud-ml-metrics).+",
```

Comment:
Nit: think this is an export artifact, not needed for the mixin in `jsonnet-libs`.

Reply:
True, will remove it, thanks!